docs/index.md

```diff
@@ -179,6 +179,30 @@ This package uses the JSON Web Token Python implementation, [PyJWT](https://gith
 
 This is the secret key used to sign the JWT. Make sure this is safe and not shared or public.
 
+Can be a scalar, a list or a dict.
+
+* When a dict, the dict keys are taken as the JWT key ids and the values as
+  keys, e.g.:
+
+  ```python
+  { "kid1": key1, "kid2": key2, ... }
+  ```
+
+  If a JWT is to be verified which contains a key id (kid header), only
+  the associated key is tried. Otherwise, or
```

Comment:
Seeing its a list, I think this _Otherwise, or_ is a bit redundant. What are your thoughts on the matter?